_overviews/scala3-migration/scaladoc-settings-compatibility.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Scaladoc settings compatibility between Scala2 and Scala3
+type: section
+description: This chapter lists all options from Scala2 and Scala3 scaladocs and explain relations between them.
+num: 24
+previous-page: options-new
+next-page:
+---
+
+The current page is stating status of scaladoc settings. The related Github issue can be found here for [discussion](https://github.com/lampepfl/dotty/issues/11907)
+
+
+| Scala2 | Scala3 | Description | Comment | Is implemented?
+| ------------- | ------------- | --- | --- | --- |
+| -doc-format | _ | Selects in which format documentation is rendered. | Actually old scaladoc supports only html, so it is in some way coherent with new scaladoc, which provides only html | <i class="fa fa-times fa-lg"></i> | 
+| -doc-title | -project | The overall name of the Scaladoc site | Aliased in [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -doc-version | -project-version | | Aliased in [#11965](https://github.com/lampepfl/dotty/issues/11965)  | <i class="fa fa-check fa-lg"></i> | 
+| -doc-footer | _ | A footer on every Scaladoc page, by default the EPFL/Lightbend copyright notice. Can be overridden with a custom footer. | Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -doc-no-compile | _ | A directory containing sources which should be parsed for docstrings without compiling (e.g. AnyRef.scala) | We don't need this as we have completely different approach to that issue using -Ydocument-synthetic-types flag for synthetic types | <i class="fa fa-check fa-lg"></i> | 
+| -doc-source-url | -source-links | A URL pattern used to link to the source file, with some variables supported... | Scala3 implementation provides richer syntax. | <i class="fa fa-check fa-lg"></i> |
```

Comment:
We should document how to migrate from Scala 2 syntax to Scala 3 syntax.

Reply:
Will add